api/hearings/hearing.mock.ts

```diff
@@ -1,15 +1,17 @@
 import MockAdapter from 'axios-mock-adapter';
-import {HttpMockAdapter} from '../common/httpMockAdapter';
-import {HEARING_ACTUAL} from './data/hearing-actuals.mock.data';
-import {HEARING_RESPONSE_RESULT} from './data/hearingResponse.mock.data';
-import {EMPTY_HEARINGS_LIST, HEARINGS_LIST} from './data/hearings.mock.data';
-import {SERVICE_HEARING_VALUES} from './data/serviceHearingValues.mock.data';
+import { HttpMockAdapter } from '../common/httpMockAdapter';
+import { HEARING_ACTUAL } from './data/hearing-actuals.mock.data';
+import { HEARING_RESPONSE_RESULT } from './data/hearingResponse.mock.data';
+import { EMPTY_HEARINGS_LIST, HEARINGS_LIST } from './data/hearings.mock.data';
+import { SERVICE_HEARING_VALUES } from './data/serviceHearingValues.mock.data';
 
 export const init = () => {
   const mock: MockAdapter = HttpMockAdapter.getInstance();
 
   const getHearingsUrl = /https:\/\/hearings.aat.service.core-compute-aat.internal\/hearings\/[0-9]{16}/;
 
+  const getHearingsActualsUrl = /https:\/\/hearings.aat.service.core-compute-aat.internal\/actuals\/[\w]*/;
+
```

Comment:
could this move down to the bottom to split with request hearing APIs? and if this can be a putHearingsActualsUrl if you want that to be different? or if you want this to be a generic could you rename this to hearingsActualsUrl if this can be used in GET/PUT/POST?

Reply:
Thank you @rayliangatos, I have now updated the code.